parts/openshift/openshiftmasterscript.sh

```diff
@@ -62,6 +71,15 @@ set -x
 routerLBHost="{{.RouterLBHostname}}"
 routerLBIP=$(dig +short $routerLBHost)
 
+# NOTE: The version of openshift-ansible for origin defaults the ansible var
+#       openshift_prometheus_node_exporter_image_version correctly as needed by
+#       origin, but for OCP it does not.
```

Comment:
This is fixed in https://github.com/openshift/openshift-ansible/commit/c27a0f46c2250ee8c58ce2f050070cd7e42a4e49, which is in >= 3.9.15.  Please add a note to that effect, and a TODO to remove this once we're installing >= 3.9.15.

Reply:
+1 - will do